Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:latest
+RUN apt-get update
+
+RUN apt-get install -y nodejs 
+RUN apt-get install -y ruby ruby-dev ruby-redcarpet
+RUN apt-get install -y zlib1g-dev liblzma-dev
+RUN apt-get install -y build-essential patch
+RUN apt-get install -y git-core
+
+RUN gem install bundler
+RUN mkdir -p /app
+
+ADD . /app/
+WORKDIR /app
+
+RUN bundle install
+
+EXPOSE 4000
+
+CMD ["bundle", "exec", "jekyll", "serve"]
```

Comment:
add newlines at the ends of files, or they aren't really text files by posix

you should configure your editor to do this, e.g. on sublime you can set the option `"ensure_newline_at_eof_on_save": true,` in your `Preferences.sublime-settings`